client/data/settings/selectors.js

```diff
@@ -233,3 +233,19 @@ export const getAdvancedFraudProtectionSettings = ( state ) => {
 export const getShowWooPayIncompatibilityNotice = ( state ) => {
 	return getSettings( state ).show_woopay_incompatibility_notice || false;
 };
+
+export const getIsStripeBillingEnabled = ( state ) => {
+	return getSettings( state ).is_stripe_billing_enabled || false;
+};
+
+export const getIsStripeBillingMigrationInProcess = ( state ) => {
```

Comment:
Should this be `getIsStripeBillingMigrationInProgress`?

```suggestion
export const getIsStripeBillingMigrationInProgress = ( state ) => {
```